packages/itwin/measure-tools/src/api/Measurement.ts

```diff
@@ -232,6 +252,26 @@ export interface MeasurementEqualityOptions {
   angleTolerance?: number;
 }
 
+export interface DrawingMetaDataProps extends Omit<DrawingMetaData, "origin" | "extents"> {
+  origin: XYProps;
+  extents?: XYProps;
+}
+
+export interface DrawingMetaData {
+  /** Id of the drawing */
+  drawingId?: string;
```

Comment:
This was a discussion with Nicholas where he mentionned the possibility that there could be other types of drawings using this in the future. For example, a runtime local drawing that does not have an ID and that extends infinitely.
The only thing that's always absolutely required (otherwise the concept would not make any sense) is the scale and origin.

Reply:
I guess that's fair, although that feels like an edge case similar to our profile/xsections. I'd consider making this required and deal with the edge case by supplying `Id64.invalid` or any other string the caller can make sense of. @bsy-nicholasw thoughts?

Reply:
I don't have a terribly strong opinion one way or another, but a lot of times when I've had an ID I usually expect it to not be `Id64.invalid` and undefined if it doesn't exist.

Reply:
Also, anything that is an `Id64String` should be typed to that and not `string`

Reply:
I would tend to go with the undefined also